decidim-admin/spec/features/manage_participatory_processes_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Manage participatory processes", type: :feature do
+  let(:admin) { create(:user, :admin) }
+  let(:participatory_process) { create(:process) }
+  let!(:participatory_process2) { create(:process) }
```

Comment:
Can't move it to a context since we'd be visiting the page before creating it, and thus it would never appear. We'd need to reload the page. Also, moving this to a before doesn't work either, so leaving as is.

The part of adding the organization is already done.